docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -246,3 +246,217 @@ Then server `$hostname` supports secure protocols that $rule `$protocols`
 ----
 Then server `vividus-test-site.herokuapp.com` supports secure protocols that contain `TLSv1.2,TLSv1.3`
 ----
+
+== JSON & JSON Path Steps
+
+:json-path: https://github.com/json-path/JsonPath#path-examples[JSON Path]
+:json-unit-options: https://github.com/lukas-krecan/JsonUnit/blob/master/README.md#options[options]
+
+
+=== *Verify context contains data* 
+
+Checks if the JSON context contains an expected data by a JSON path
+
+[source,gherkin]
+----
+Then JSON element by JSON path `$jsonPath` is equal to `$expectedData`$options
+----
+
+* `jsonPath` - {json-path}
+* `expectedData` - expected value
```

Comment:
`expected JSON`

Reply:
https://github.com/vividus-framework/vividus/pull/1150#discussion_r537778273

Reply:
that's ok, `JSON` could be any of this list, so `JSON` is better naming since it limits the variety, but `value` is anything